vllm/model_executor/layers/attention/backends/flash_attn.py

```diff
@@ -99,19 +100,33 @@ def forward(
                 # normal attention
                 # When block_tables are not filled, it means q and k are the
                 # prompt, and they have the same length.
-                output = flash_attn_varlen_func(
-                    q=query,
-                    k=key,
-                    v=value,
-                    cu_seqlens_q=input_metadata.seq_start_loc,
-                    cu_seqlens_k=input_metadata.seq_start_loc,
-                    max_seqlen_q=input_metadata.max_seq_len,
-                    max_seqlen_k=input_metadata.max_seq_len,
-                    softmax_scale=self.scale,
-                    causal=True,
-                    window_size=self.sliding_window,
-                    alibi_slopes=self.alibi_slopes,
-                )
+                if is_hip():
+                    # window_size and alibi_slopes not supported
```

Comment:
nice catch! btw, what's happening if you pass window_size and alibi_slopes in this case? Does it just break?

Reply:
The `Dockerfile.rocm` uses this repo to install flash-attention: https://github.com/ROCm/flash-attention
This API `flash_attn_varlen_func` in this flash-attention repo currently does not contain these two parameters . you would get error if you pass the parameters like :
```
TypeError: flash_attn_varlen_func() got an unexpected keyword argument 'window_size' 
```